circle.yml

```diff
@@ -0,0 +1,21 @@
+machine:
+  xcode:
+    version: "8.0"
+
+dependencies:
+  pre:
+    - brew update 1> /dev/null 2> /dev/null
+    - brew outdated carthage || brew upgrade carthage
+
+checkout:
+  post:
+    - git submodule update --init --recursive
+test:
+  override:
+    - ./scripts/macOS
```

Comment:
#pp I think the build script is simple enough that we don't need to introduce a dependency on fastlane.